integration_tests/index.d.ts

```diff
@@ -1,9 +1,9 @@
 declare namespace Cypress {
-  interface Chainable {
+  interface Chainable<Subject = unknown> {
```

Comment:
intellij was moaning about `Chainable` on line 7 not being generic, so fixes the warning